job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/DBConnection.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016-2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.scheduled.executor;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.text.MessageFormat;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DBConnection
+{
+    private static final Logger LOG = LoggerFactory.getLogger(DBConnection.class);
+    
+    private static final String JDBC_POSTGRESQL_PREFIX = "jdbc:postgresql:";
+    private static final String JDBC_DRIVER = "org.postgresql.Driver";
+    
```

Comment:
This is actually a utility class so it should have an empty private default constructor (to prevent accidental instantiation).